### PR TITLE
Add DBRS dockerfiles

### PR DESCRIPTION
**Pull request by gabrielbriones**
This commit includes the dockerfiles and documentation for Database Reference Stack v0.1.0, they will be located under stacks/dbrs directory. This version includes Redis and Cassandra applications.

Each application includes the scripts and conf directores with the files required for building the container.

For Cassandra there is a helm chart, and for Redis there is a yaml configuration file for a redisfailover instance. These files are included for an easier adoption as a kubernetes application.

Signed-off-by: Gabriel Briones <gabriel.briones.sayeg@intel.com>

### CODE REVIEW

File: stacks/dbrs/cassandra/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
+FROM clearlinux/stacks-clearlinux:latest
+MAINTAINER otc-swstacks@intel.com
+
+RUN swupd bundle-add curl java-runtime python2-basic which pmdk sudo
```
Comment (bryteise) [inline stacks/dbrs/cassandra/Dockerfile]: Ah this reminds me of a general problem that needs to be scrubbed but if this requires python2 then I would -1. python2 is going to be removed from the distro completely in a few months so I don't want any more new users of it.

File: stacks/dbrs/redis/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM clearlinux AS build-redis
+MAINTAINER otc-swstacks@intel.com
+
+RUN swupd bundle-add --quiet --no-progress git c-basic devpkg-ndctl os-testsuite-phoronix-server
```
Comment (bryteise) [inline stacks/dbrs/redis/Dockerfile]: os-testsuite-phoronix-server seems odd to be adding. Why are is it being pulled in?

Comment (gabrielbriones) [reply stacks/dbrs/redis/Dockerfile]: We require libnuma, this is the lowest size bundle containing it.

Comment (gabrielbriones) [reply stacks/dbrs/redis/Dockerfile]: @aduejazz is there any other package required from os-testsuite-phoronix?

Comment (aduejazz) [reply stacks/dbrs/redis/Dockerfile]: unfortunately we only need libnuma.so, and iirc that's the smallest bundle containing that library.

File: stacks/dbrs/cassandra/Dockerfile (hunk 1 above)
Comment (gabrielbriones) [reply stacks/dbrs/cassandra/Dockerfile]: There is a reported unresolved issue related to the Cassandra client depending on python 2: https://issues.apache.org/jira/browse/CASSANDRA-10190.

Comment (bryteise) [reply stacks/dbrs/cassandra/Dockerfile]: Ah that's currently not working today in Clear then. Would you be able to submit the work in progress patches to our cassandra package for now? It seemed like from that thread the patches are working just some small nit picks and test case issues.

Comment (gabrielbriones) [reply stacks/dbrs/cassandra/Dockerfile]: We are not using Cassandra from ClearLinux, it's a Cassandra version from pmem team. We are also planning to get rid of python2 in a next release.

File: stacks/dbrs/redis/Dockerfile (hunk 2 above)
Comment (bryteise) [reply stacks/dbrs/redis/Dockerfile]: Ah! I created a numa devpkg that'll show up next week.

Comment (aduejazz) [reply stacks/dbrs/redis/Dockerfile]: thanks, really appreciated!

File: stacks/dbrs/cassandra/Dockerfile (hunk 1 above)
Comment (bryteise) [reply stacks/dbrs/cassandra/Dockerfile]: Ah okay. When is the next release?

Comment (aduejazz) [reply stacks/dbrs/cassandra/Dockerfile]: so our first release is planned to happen next monday (hopefully) and we can plan our next release to happen anytime, probably after getting rid of this pyhon2 dependency and incorporating the new bundle you created for numa (and get rid of os-testsuite-phoronix-server)

Comment (bryteise) [reply stacks/dbrs/cassandra/Dockerfile]: Okay, seems like a decent path to not breaking.

**Pull request merged**